content-scripts/src/modules/login.js

```diff
@@ -0,0 +1,82 @@
+import { getStorage, setStorage } from "./utilities/storage";
+
+const emptyLogin = { autoLogin: { active: true, verifed: false, user_info: "" } };
+
+
+export const rememberLogin = async () => {
+    var auto_login = await getStorage("autoLogin");
+    if (auto_login === undefined) {
+        await setStorage(emptyLogin);
+        auto_login = emptyLogin;
+    }
+    if (auto_login.active == true) {
+        console.log(auto_login);
+        if (auto_login.user_info == "") {
+            if (document.getElementsByClassName("autenticado").length === 0) {
+                //inject button onClick
+                document
+                    .querySelector(".autenticacao > form:nth-child(2) > button")
+                    .addEventListener("click", (event) => {
+                        event.preventDefault();
+                        auto_login.user_info = btoa(JSON.stringify({
+                            user: document.getElementById("user").value,
+                            pass: document.getElementById("pass").value
+                        }));
+
+                        tryLogin(auto_login).then(
+                            async (res) => {
+                                if (res.status != 200) {
+                                    console.log("Something went wrong while logging in...");
+                                    return;
+                                }
+                                const loggedIn = await chrome.runtime.sendMessage({ type: "login", auto_login: auto_login });
+                                 if(loggedIn === false){
+                                    //TODO: (issue #59) show error to user
+                                    console.log("Wrong details... try again");
+                                    return;
+                                }
+                            }
+                        );
+                    })
```

Comment:
We can make this listener async, removing the need for another callback in `then`